### PR TITLE
Enable to import entries belonging to multiple entities

### PR DESCRIPTION
**Pull request by syucream**
It enables to import entries belonging to multiple entities that we can get with exporting advanced search result, on 1 import job.
For now, we can submit an import job on advanced search result page.

image

### CODE REVIEW

File: templates/advanced_search/result.html [hunk 1]
```diff
@@ -25,6 +25,7 @@ <h4>
           </button>
           <button type="button" class="btn btn-primary btn-sm export_yaml">YAML 出力</button>
           <button type="button" class="btn btn-primary btn-sm export_csv">CSV 出力</button>
+          <a class="btn btn-secondary btn-sm" href="/dashboard/import_search_result/">インポート</a>
```
Comment (userlocalhost) [inline templates/advanced_search/result.html]: You may want to put the entry point of this feature at existing place.
User might be used to do this operation at ordinary import way from Entry list page.

**Review by userlocalhost — COMMENTED** [review]
Thank you very much for implementing feature that users desire to!
But, I'm not sure whether we should make another new UI entry-point for importing. Because there are many ways to import information (from the Entity list page and individual Entry list pages.
Anyway, this must be very useful feature. We are very much looking forward to merging this into this application.
Thank you 👍

File: templates/advanced_search/result.html (hunk 1 above)
Comment (userlocalhost) [reply templates/advanced_search/result.html]: I changed my mind. It's OK 👍

**Comment by syucream** [comment]
@userlocalhost to simplify importing features, I've unified importing multiple entities on entry import task. How about this?

**Review by hinashi — APPROVED** [review]
LGTM!

**Review by userlocalhost — APPROVED** [review]
LGTM 👍 Thank you for your hard work!! It's a great job!!

**Comment by userlocalhost** [comment]
Would you please adding line that describes this feature at the CHANGELOG?

**Pull request merged**